src/components/itemMediaInfo/itemMediaInfo.js

```diff
@@ -84,6 +84,7 @@ function getMediaSourceHtml(user, item, version) {
             case 'Data':
             case 'Subtitle':
             case 'Video':
+            case 'Lyric':
                 translateString = stream.Type;
                 break;
```

Comment:
Not requesting any changes, just wondering why not
```js
let translateString = stream.Type;
if (stream.Type === 'EmbeddedImage') translateString = 'Image'; // `switch` may be used as well
```

Reply:
I originally did that change but reverted when I realized there was localization.
This way we are forced to add localization :smile: